src/ui/wxWidgets/PasswordSafeFrame.cpp

```diff
@@ -684,6 +706,25 @@ void PasswordSafeFrame::CreateMenubar()
   // Update menu selections
   GetMenuBar()->Check((IsTreeView()) ? ID_TREE_VIEW : ID_LIST_VIEW, true);
   GetMenuBar()->Check(PWSprefs::GetInstance()->GetPref(PWSprefs::UseNewToolbar) ? ID_TOOLBAR_NEW : ID_TOOLBAR_CLASSIC, true);
+  UpdateTreeSortMenu();
+}
+
+/**
+ * Update check for sorting and enable or disable the menu bar for orting
+ */
+void PasswordSafeFrame::UpdateTreeSortMenu()
+{
+  auto menuBar = GetMenuBar();
+  
+  menuBar->Check(ID_SORT_TREE_BY_GROUP, IsTreeSortGroup() ? true : false);
+  menuBar->Check(ID_SORT_TREE_BY_NAME, IsTreeSortName() ? true : false);
+  menuBar->Check(ID_SORT_TREE_BY_DATE, IsTreeSortDate() ? true : false);
```

Comment:
Wouldn't it be sufficient to use the return value of the function directly?

Reply:
Although the logic is better readable with the explicit values I followed the demanded change.